.gitignore

```diff
@@ -16,3 +16,4 @@ _plugins
 _tdeps
 logs
 /_build
+ebin/
```

Comment:
can you add an empty line at the end of the file?

Reply:
Fixed